packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -262,7 +262,8 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
   }
 
   bool _shouldStart(ScrollNotification notification) {
-    return (notification is ScrollStartNotification || (notification is ScrollUpdateNotification && notification.dragDetails != null && widget.triggerMode == RefreshIndicatorTriggerMode.anywhere))
+    return ((notification is ScrollStartNotification && notification.dragDetails != null)
```

Comment:
This is getting more and more mysterious, can you add some comments explain what does it mean when `notification.dragDetails != null` or define boolean getter with a more descriptive name.